src/main/java/com/Others/FastPower.java

```diff
@@ -0,0 +1,37 @@
+package src.main.java.com.Others;
+
+import java.math.BigInteger;
+
+/**
+ * We may calculate power with loops, but what if the index is too large ?
+ * FastPower aims to calculate quickly in this circumstances with time complexity O(log k),
+ * where k is the index.
+ *
+ * @author DDullahan
```

Comment:
Please do not add a signature `@author xxx` inside the code. See [Contribution Guidelines](https://github.com/TheAlgorithms/Java/tree/Development#contribution-guidelines) for details.